src/snowflake/connector/config_manager.py

```diff
@@ -109,8 +113,15 @@ def value(self) -> Any:
         source = "environment variable"
         loaded_env, value = self._get_env()
         if not loaded_env:
-            source = "configuration file"
-            value = self._get_config()
+            try:
+                value = self._get_config()
+                source = "configuration file"
+            except MissingConfigOptionError:
+                if self.default:
+                    source = "default_value"
```

Comment:
We talked this through offline, since this value is truly unused for now. The reason why I added it originally is to include the source in logging messages. We'll leave it in for now and make a decision about it in the future.